app/src/test/java/io/intrepid/contest/screens/sendinvitations/SendInvitationsPresenterTest.java

```diff
@@ -0,0 +1,108 @@
+package io.intrepid.contest.screens.sendinvitations;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import io.intrepid.contest.R;
+import io.intrepid.contest.testutils.BasePresenterTest;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
Shouldn't need the runwith annotation or the `MockitoAnnotations.initMocks(this)` method. Since you're extending `BasePresenterTest`, that takes care of initializing the mocks with the `MockitoRule` that is defined there. Also, if you ever need to use either of the above to methods, you only need one or the other; shouldn't need to have both.